src/NzbDrone.Core/Download/TorrentClientBase.cs

```diff
@@ -216,7 +216,7 @@ private string DownloadFromMagnetUrl(RemoteEpisode remoteEpisode, string magnetU
             {
                 _logger.Error(ex, "Failed to parse magnetlink for episode '{0}': '{1}'", remoteEpisode.Release.Title, magnetUrl);
 
-                return null;
+                throw new ReleaseDownloadException(remoteEpisode.Release, "Failed to parse magnetlink for episode '{0}': '{1}'", remoteEpisode.Release.Title, magnetUrl);
```

Comment:
Can this include the caught exception?

I think the error logging should be removed as well so it's not double logged.